A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_PerformancePage.js

```diff
@@ -482,8 +482,8 @@ class CDUPerformancePage {
             titleColor = "green";
         }
         let qnhCell = "[ ]";
-        if (isFinite(mcdu.perfApprQNH)) {
-            qnhCell = mcdu.perfApprQNH.toFixed(0);
+        if (isFinite(mcdu.perfApprQNH) | /[0-9]{2}.[0-9]{2}/.test(mcdu.perfApprQNH)) {
```

Comment:
1. can you store that regex in a constant at the top of the file ?

2. it's `||` not `|`. Did you test this ?

Reply:
I'll change the regexes. It did work when testing but I'll switch to the real or operator.